### PR TITLE
Truncate model configuration name in plots

### PR DESCRIPTION
**Pull request by pskiran1**

### CODE REVIEW

File: model_analyzer/plots/simple_plot.py [hunk 1]
```diff
@@ -184,6 +186,35 @@ def plot_data_and_constraints(self, constraints):
         self._ax.legend()
         self._ax.grid()
 
+    def truncate_model_config_name(self, model_config_name):
```
Comment (nv-braf) [inline model_analyzer/plots/simple_plot.py:189]: I can see this being used outside of simple plots. Can you make a new file called `report_utils.py` and add this?

Comment (nv-braf) [reply model_analyzer/plots/simple_plot.py:189]: We also need unit testing here, so I would add a test_report_utils.py and add testing.

**Review by nv-braf — CHANGES_REQUESTED** [review]

File: model_analyzer/plots/simple_plot.py (hunk 1 above)
Comment (pskiran1) [reply model_analyzer/plots/simple_plot.py]: Done the changes.

**Review by nv-braf — APPROVED** [review]
Looks great!

**Pull request merged**